tools/template/cartesi-machine.template

```diff
@@ -2,4 +2,4 @@
 export LUA_PATH_5_4="${LUA_PATH_5_4:-;};ARG_LUA_PATH"
```

Comment:
I started to have some issues regarding multiple installed versions of cartesi machine in my environment.
For example I have a cartesi-machine machine installed at `/usr` and another one in `/opt/cartesi`, when I execute `/opt/cartesi/bin/cartesi-machine` the `LUA_PATH_5_4` tries to look up first the installation at `/usr`, instead of the one at `/opt/cartesi`, inverting the order here should fix this problem. That is, place `ARG_LUA_PATH` before:

```bash
export LUA_PATH_5_4="ARG_LUA_PATH;${LUA_PATH_5_4:-;}"
```

And then we should do this for all templates.

Reply:
I made this in https://github.com/cartesi/machine-emulator/pull/114